site/_data/master-toc.yml

```diff
@@ -10,7 +10,7 @@ toc:
   - title: Install
     subfolderitems:
       - page: Overview
-        url: /install-overview
+        url: /getting-started
```

Comment:
If we're going to split out the advanced/customize items to their own page, we probably want to add that new page to the TOC as well

Reply:
I split it. But not sure if my change actually works.

Reply:
it does!